backend/src/game/dice_handler.py

```diff
@@ -0,0 +1,24 @@
+import backend.src.game.dice as dice
+import backend.src.game.constants as constants
+
+
+class DiceHandler(object):
```

Comment:
There's no reason in Python3 to inherit from object, see: https://stackoverflow.com/questions/4015417/python-class-inherits-object

Reply:
Thanks for including the references to read, it's been really helpful